ksql-docker/pom.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2018 Confluent Inc.
+  ~
+  ~ Licensed under the Confluent Community License (the "License"); you may not use
+  ~ this file except in compliance with the License.  You may obtain a copy of the
+  ~ License at
+  ~
+  ~ http://www.confluent.io/confluent-community-license
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  ~ WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>ksql-parent</artifactId>
+    <groupId>io.confluent.ksql</groupId>
+    <version>5.5.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>ksql-docker</artifactId>
+
+  <properties>
+    <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+    <main-class>io.confluent.ksql.rest.server.KsqlRestApplication</main-class>
+    <cli.skip-execute>false</cli.skip-execute>
+    <cli.main-class>${main-class}</cli.main-class>
+    <docker.skip-build>${skip.docker.build}</docker.skip-build>
+    <docker.skip-test>${skip.docker.test}</docker.skip-test>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.confluent.ksql</groupId>
+      <artifactId>ksql-rest-app</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>io.confluent.ksql</groupId>
+      <artifactId>ksql-cli</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>io.confluent.ksql</groupId>
+      <artifactId>ksql-functional-tests</artifactId>
+      <version>${project.version}</version>
```

Comment:
do we need to specify the version here?

Reply:
We do, else maven errors out with 
```
[ERROR]   The project io.confluent.ksql:ksql-docker:5.5.0-SNAPSHOT (/Users/vpapavasileiou/Documents/confluent/ksql/ksql-docker/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for io.confluent.ksql:ksql-functional-tests:jar is missing. @ line 50, column 17
```

Reply:
Ahhhh.... instead of updating it here, can you update it in the parent POM alongside the other cross-submodule dependencies?

Reply:
done